tests/core/jobs/test_playtime_entries.py

```diff
@@ -229,6 +230,28 @@ def test_do_run(self, db: DatabaseTransactionFixture):
         collection2 = db.collection()
         library2 = db.library()
 
+        isbn_ids: dict[str, Identifier] = {
+            "i1": db.identifier(
+                identifier_type=Identifier.ISBN, foreign_id="080442957X"
+            ),
+            "i2": db.identifier(
+                identifier_type=Identifier.ISBN, foreign_id="9788175257665"
+            ),
+        }
+        identifier.equivalencies = [
+            Equivalency(
+                input_id=identifier.id, output_id=isbn_ids["i1"].id, strength=0.5
+            ),
+            Equivalency(
+                input_id=isbn_ids["i1"].id, output_id=isbn_ids["i2"].id, strength=1
+            ),
+        ]
+        strongest_isbn = isbn_ids["i2"].identifier
+        no_isbn = ""
+
+        # We're using the RecursiveEquivalencyCache, so must refresh it.
+        EquivalentIdentifiersCoverageProvider(db.session).run()
```

Comment:
While this is the pristine method of creating the Recursive cache, for the purpose of the test we can just create RecursiveEquivalencyCache entries directly, since the coverage provider is not the target of the test.
Eg.
```
create(db, RecursiveEquivalencyCache, parent_id=1, identifier_id=2)
create(db, RecursiveEquivalencyCache, parent_id=1, identifier_id=3)
```

Reply:
I have mixed feelings about this. On the one hand, I don't want to conflate the testing of different parts of the system too much; on the other hand, I don't want to get bogged down in the details of how the RecursiveEquivalencyCache optimization is set up in every test that uses it (I expect it to get more adoption).